### PR TITLE
An optional title attribute for links and paths

### PR DESCRIPTION
**Pull request by statico**
Sometimes our developers export things from Xcode with names other than "Foo.app," such as "Foo-2.5.1-test.app." In master, that filename gets copied to the disk image. This fix lets us rename things once they're on the image.

Thanks for the great module!

### CODE REVIEW

File: lib/appdmg.js [hunk 1]
```diff
@@ -387,7 +387,8 @@ module.exports = exports = function (options) {
     ds.setWindowSize(global.bkgsize[0], global.bkgsize[1])
 
     global.opts.contents.forEach(function (e) {
-      ds.setIconPos(path.basename(e.path), e.x, e.y)
+      var p = typeof e.title === 'undefined' ? path.basename(e.path) : e.title
+      ds.setIconPos(path.basename(p), e.x, e.y)
```
Comment (LinusU) [inline lib/appdmg.js]: path.basename is called twice when not using title. Does it make sense to call `basename` on `title`? I don't think we should allow `/` in the `title`.

**Comment by LinusU** [comment]
Looks good, would you mind adding documentation as well?

File: lib/appdmg.js (hunk 1 above)
Comment (statico) [inline lib/appdmg.js]: Re: path.basename, whoops!

Re `/`, do you mean add a check to see if `title` is a valid filename?

**Comment by eugirdor** [comment]
Any changes of this getting merged?

**Comment by LinusU** [comment]
Fixed by #110, released as 0.4.5

**Pull request closed without merging**